### PR TITLE
Update CHANGELOG for v2.3.0

### PR DESCRIPTION
**Pull request by andschwa**

### CODE REVIEW

File: PowerShellEditorServices.Common.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.3.0</VersionPrefix>
-    <VersionSuffix>preview.4</VersionSuffix>
+    <VersionPrefix>2.4.0</VersionPrefix>
+    <VersionSuffix>preview</VersionSuffix>
```
Comment (rjmholt) [inline PowerShellEditorServices.Common.props:4]: I think you just want to remove this entry no?

Comment (rjmholt) [reply PowerShellEditorServices.Common.props:4]: Unless this is the post-release bump, in which case I'd call it `preview.1`

Comment (andschwa) [reply PowerShellEditorServices.Common.props:4]: No, this is the commit after the tag we're back to 2.4.0-preview.

Comment (rjmholt) [reply PowerShellEditorServices.Common.props:4]: I don't follow -- are you saying it's not the post-release version bump?

In any case, we've never had a prerelease tag that didn't match `preview\.\d+`

Comment (andschwa) [reply PowerShellEditorServices.Common.props]: Oi, our messages were crossed. Fixed.

**Comment by andschwa** [comment]
Ping @rjmholt for a review.

**Review by rjmholt — APPROVED** [review]

**Pull request merged**